packages/fxa-payments-server/src/routes/Subscriptions/index.stories.tsx

```diff
@@ -51,6 +51,24 @@ function setupVariantStories(
         }}
       />
     ))
+    .add('subscribed with web subscription with coupon', () => (
+      <SubscriptionsRoute
+        routeProps={{
+          ...subscribedProps,
+          subsequentInvoices: {
```

Comment:
Could we add this to `subscribedProps` instead?

Reply:
done